### ISSUE TITLE
alienwawre17r4 ubuntu16.04

### ISSUE THREAD

**Issue by mingrui** [awesomebytes/alienware15r3_ubuntu14#4]
I followed everything, but this line of code gave me a black screen on reboot:

`sudo prime-select intel`

I rebooted into recovery mode and tried switching back to nvidia, allowed me to reboot to X again:

`sudo prime-select nvidia`

running the cuda samples I don't have to append with

`optirun`

I think maybe this is because after I installed ubuntu, I used the graphical interface menu: additional drivers, to switched to nvidia gpu.

But anyways switching to nvidia worked for me.

Can I leave this here in case other people run into the same problem?
Thanks for your write up!

**Comment by mingrui** [comment]
UPDATE: 
I think the reason is because my alienware17r4 gtx1080, the screen is the 120hz g-sync panel, reading alienware's docs, this configuration doesn't have nvidia optimus installed.

**Comment by HuaizhengZhang** [comment]
@mingrui Hey. Have you successfully installed by following this instruction? I just got an Alienware with GPU 1080 and I want to try this instruction. Can you give me some guide?

**Comment by dkorobov2** [comment]
Thank you for this!

**Comment by mingrui** [comment]
@HuaizhengZhang Yes I am running ubuntu right now on my alienware 17 r4 1080 with the 120hz panel! follow this guide!

**Comment by d0x90** [comment]
Hello @mingrui @HuaizhengZhang @dkorobov2 @awesomebytes , do you have problems with the duration of your battery?  average of 2 hours,  Windows 6 hours.
Running ubuntu 18 
Alienware 17 R4 gtx1070

**Comment by awesomebytes** [comment]
Hello. Even tho your model is different so I can't assure the same results, what I do is... Disable the Nvidia card to save battery in Ubuntu when I need to. I almost never do it so I don't really know what's the battery duration.

**Comment by d0x90** [comment]
Hi @awesomebytes  thanks for your quick answer
I already tried that.
sudo prime-select intel
but same result. :sad:

**Comment by awesomebytes** [comment]
Unfortunately the battery saving goodies usually only work in Windows. Specially with Nvidia involved in the game. You should make sure your Nvidia card is disabled tho.

In my laptop I can disable the Nvidia card in Windows with a hotkey. If I go back to Ubuntu afterwards the Nvidia card is completely disabled, so I'm sure (as much as I can be) it's not sucking battery. You could give it a try if you also have that hotkey/function. It's very annoying but better than nothing.

**Comment by d0x90** [comment]
Thanks @awesomebytes 
I will try it and tell you how it was.

**Comment by mingrui** [comment]
@newoverflow On my 17r4 with GTX1080 and 120hz screen, I can barely get 2 hours in Ubuntu, because nvidia optimus doesn't work for 120hz screen models.

**Comment by d0x90** [comment]
@mingrui  @awesomebytes  Oh, noob question but, does it directly affect my battery life ? How fast?

**Comment by awesomebytes** [comment]
@newoverflow Well, a battery has a limited number of cycles and loses efficiency over time... so if you keep emptying and charging the battery, of course, it will affect the battery life. Note that if you keep it plugged the battery stays healthy (the charging electronics should take care of not damaging your battery).

Btw in a quick google some people say that your model may have in the BIOS an option to disable either the intel or the nvidia GPU (so the OS won't even see it). You could check that out. They call it _PEG_. Altho some say it's the same than using the hotkey I mentioned in Windows. Good luck!

**Comment by mingrui** [comment]
@newoverflow alienware has many replacement parts for sale on alibaba / taobao (I live in China and purchased replacement heatsink before), including batteries. They are relatively easy to take apart and replace. So if you need a replacement that's not a big problem.
